FeedStoreChallenge/ManagedCache.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ManagedCache.swift
+//  FeedStoreChallenge
+//
+//  Created by Joshua Bryson on 9/9/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import CoreData
+
+@objc(ManagedCache)
+internal class ManagedCache: NSManagedObject {
```

Comment:
This class is not intended to be inherited thus it should be `final`
Please make the class `final`.
```suggestion
final class ManagedCache: NSManagedObject {
```